stl/inc/expected

```diff
@@ -190,6 +190,21 @@ class expected {
     template <class _UTy, class _UErr>
     friend class expected;
 
+    template <class _Uty, class _UErr>
+    static constexpr bool _Allow_unwrapping = disjunction_v<is_same<remove_cv_t<_Ty>, bool>,

```

Comment:
We conventionally avoid relying on `class`'s default of `private` access control. Here, `class expected` began with only `static_assert`s and `friend`, neither of which actually care about access control, so as a special exception we don't bother to say `private:`. However, by moving `_Allow_unwrapping` up, an actual thing is being defined here, so we should say `private:`.